Blob tweaks and a new blob chemical

ChangelingRain opened the pull request:
:cl: Joan
experiment: Blob expansion is faster near the expanding blob, but slower further away from it.
tweak: Blob spores produce a slightly larger smoke cloud when dying. Sporing Pods spores and blob zombies produce the current small cloud.
rscdel: Blobbernauts can no longer pull anything at all.
tweak: Blobbernauts health reduced to 200, but blobbernauts now take half brute damage.
tweak: Blobbernauts now take massive damage from fire.
tweak: Blobbernauts do approximately 3 more damage when attacking.
wip: Replicating Foam will expand more actively when damaged.
rscadd: Adds Electromagnetic Web, which is light blue, does burn damage and EMPs targets, and causes a small EMP when dying.
rscadd: Electromagnetic Web takes more damage; a normal blob that hasn't been near a node or the core for 6~ seconds can be oneshotted by a laser.
/:cl:

Tweaks Ripping Tendrils color to be lighter and Energized Fibers colour to be darker. Not a lot, though!
Fixes an edge case where the blobbernaut power could destroy factories.
Adds some sound feedback to making a blobbernaut.

ChangelingRain committed:
Tweaks blob expansion to be faster near the expanding blob but slower near the edges of the blob
Blob spores now produce a slightly larger cloud of smoke when dying. Fragile spores and blob zombies produce the current, smaller cloud.

Replicating Foam will try to expand when damaged much more often.
Adds Electromagnetic Web, which does burn damage and EMPs targets, as well as causing a small EMP when dying, but takes more damage; a normal blob that hasn't been near a node or the core for 6~ seconds can be oneshotted by a laser.

ChangelingRain committed:
correct arg and var names, ho

WJohn commented:
C-could you nerf blobbernauts while you're at it? People still spam the absolute shit out of blobbernauts and they're still a ridiculously easy guaranteed victory.

Please lower their health, 240 HP is insanity. It gives them free reign to be as reckless as they want and drag humans back to the blob.

PKPenguin321 commented:
tbh i really wanna personally revert player blobbernauts so you'd really ought to do what he says before i do something drasitc mr changelingrain

ChangelingRain committed:
Merge branch 'master' of https://github.com/tgstation/-tg-station into fixedblobchemicals

ChangelingRain committed:
Blobbernauts can no longer pull.
Decreased blobbernaut health to 200.
Blobbernauts take half brute damage.
Blobbernauts do around 3 more damage with chemical attacks.

ChangelingRain committed:
weakens death EMP
